gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/io/binary/DataType.java

```diff
@@ -44,8 +44,8 @@ public enum DataType {
     VERTEX(0X11),
     VERTEXPROPERTY(0X12),
     BARRIER(0X13),
-    BINDING(0X14),
```

Comment:
this change begs an update to the IO docs and what will be the start of the upgrade docs. i don't know if this doc work is being tracked separately but i think subtle things will be missed if it all waits until the master-http branch merge.